README-comments.md

```diff
@@ -0,0 +1,268 @@
+
+A short time after you've initiated the cluster creation, the web interface shows instructions for downloading the `bx` CLI, installing the container-service plugin into it, logging into your account ... all the way to configuring `kubectl` with the cluster and starting a local proxy to show the Kubernetes Dashboard for the cluster you've created. Like this:
+
+![Kubernetes Dashboard](images/KubeDash.png)
+
+## Somewhere to put Docker images
+When you run you microservices on Kubernetes, Kubernetes deploys Docker containers made from Docker images which it retrieves from a Docker registry. Your IBM Cloud account has a private Docker registry associated with served up by the IBM Cloud Container Registry. To start with your private registry is empty.
+
+The [Container Registry Quick Start](https://console.bluemix.net/containers-kubernetes/registry/start) gives instructions for installing the container-registry plugin to the `bx` CLI and how to create a namespace which can receive images pushed to it. This will come in useful later when we use Lightbend Orchestration for Kubernetes to push the Chirper application images to the registry.
+
+## Developing Chirper with the IBM Cloud
+You may have already followed [the instructions for setting up the Chirper example](https://github.com/longshorej/lagom-java-chirper-tooling-example/blob/master/README.md) which includes instructions specific to Minikube. The set of prequisites and steps are similar for deploying to IBM Cloud:
+
+Install the following programs on your system:
+
+* [Docker](https://store.docker.com/search?type=edition&offering=community)
+* [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+* [SBT](http://www.scala-sbt.org/)
+* [reactive-cli](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/#install-the-cli)
+
+Set up the Reactive Sandbox instance of Cassandra in the Container Service:
+
+The [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox) includes development-grade (i.e. it will lose your data) installations of Cassandra, Elasticsearch, Kafka, and ZooKeeper. It's packaged as a Helm chart for easy installation into your Kubernetes cluster.
+
+```bash
+helm init --upgrade
+helm repo add lightbend-helm-charts https://lightbend.github.io/helm-charts
+helm update
+```
+
+Verify that Helm is available (this takes a minute or two):
+
+```bash
+kubectl --namespace kube-system get deploy/tiller-deploy
+```
+
+```
+NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+tiller-deploy   1         1         1            1           3m
+```
+
+Install the sandbox. Since Chirper only uses Cassandra, we're disabling the other services but you can leave them enabled by omitting the `set` flag if you wish.
```

Comment:
I know we had this in the readme, but maybe it's simpler to just omit this and install the sandbox with default settings?

Reply:
You mean change:

```helm install lightbend-helm-charts/reactive-sandbox --name reactive-sandbox --set elasticsearch.enabled=false,kafka.enabled=false,zookeeper.enabled=false```

to

```helm install lightbend-helm-charts/reactive-sandbox --name reactive-sandbox```

and adjust the output coming from:

```kubectl get deploy/reactive-sandbox```

I'm concerned our (really quite small) Kube node is going to get overloaded with this.